website/docs/policy/weave-policy-profile.mdx

```diff
@@ -37,13 +37,12 @@ resources: # specifies the path to each required policy
 
 The profile then needs to be configured with the necessary config to be able to reach the repository that is acting as a policy library.
 
-```yaml
-policyLibraryConfig:
-  url: ssh://git@github.com/myorg/policy-library # repo url
-  branch: agent-profile # which branch to use
-  kustomizationPath: ./ # directory containing the kustomization file
-  private: true # set to true if a private repo
-  secretRef: policy-library-auth # secret containing cred to authenticate with a private repo
+```
+policySource:
```

Comment:
Should this also be namespaced under `policy-agent`?

Reply:
I think it makes more sense for them to be separate, since policy source is its own concept and its values have no bearings on the agent chart at all